data_structure/linked_list.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
```

Comment:
@mariangusatu 
Could you point out exact lines where code was changed as because of rename, code changes are not visible.